.github/workflows/deploy-artifact.yml

```diff
@@ -72,6 +73,8 @@ jobs:
         run: |
           if ! [ -z "${{ github.event.release.tag_name }}" ] && ! grep -q "$TAG" <<< "SNAPSHOT"; then
```

Comment:
💭 the second condition might be obsolete?

Reply:
I think so yeah, I'll remove it and mere